addons/website/models/ir_http.py

```diff
@@ -66,6 +66,17 @@ def routing_map(self, key=None):
             key = request.website_routing
         return super().routing_map(key=key)
 
+    @classmethod
+    def _slug(cls, value: models.BaseModel | str) -> str:
+        try:
+            if value.id and value.seo_name:
+                name = value.seo_name
+                slugname = name and cls._slugify(name).strip().strip('-')
+                return f"{slugname}-{value.id}" if slugname else str(value.id)
```

Comment:
```suggestion
            if value.id and value.seo_name:
                slugname = cls._slugify(value.seo_name).strip().strip('-')
                return f"{slugname}-{value.id}"
```
maybe add `.strip().strip('-')` at the `if` if you really want to make sure it is fine